espnet/nets/pytorch_backend/e2e_mt_transformer.py

```diff
@@ -461,8 +446,6 @@ def translate(self, x, trans_args, char_list=None, rnnlm=None, use_jit=False):
                     remained_hyps.append(hyp)
 
             # end detection
-            from espnet.nets.e2e_asr_common import end_detect
-
             if end_detect(ended_hyps, i) and trans_args.maxlenratio == 0.0:
```

Comment:
This is not related to this PR.

I'm just curious whether this `end_detect` is working well for MT.
The [hyper-parameters](https://github.com/espnet/espnet/blob/6f6452a3891d11b0b0306084dd46a464ddf8e25c/espnet/nets/e2e_asr_common.py#L19) in the `end_detect` function were determined by ASR.
I think they are based on a similar problem and we could use the same hyper-parameters though.
@takaaki-hori, what do you think?